tla/consensus/Traceccfraft.tla

```diff
@@ -135,6 +135,7 @@ IsDropPendingTo ==
             /\ msg.type = RaftMsgType[logline.msg.packet.msg]
             /\ msg.dest = logline.msg.to_node_id
             /\ msg.source = logline.msg.from_node_id
+            /\ msg.term = logline.msg.packet.term
```

Comment:
I would suggest to add operators like `Traceccfraft!IsAppendEntriesRequest` for each message type instead of one generic "handle" to strengthen validation.  Then, use the operators like in:

```tla
 Network!DropMessage(logline.msg.to_node_id, LAMBDA msg: IsAppendEntriesRequest(msg, logline.msg.to_node_id, logline.msg.from_node_id, logline))
```

Additionally, instead of asserting that at least one message is dropped, we could assert that exactly one message gets dropped.  In other words, we can add `Network!OneLessMessage` next to `Network!OneMoreMessage`.